assets/js/a11y.js

```diff
@@ -0,0 +1,275 @@
+/*!
+ * Copyright (c) 2022, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// "config" object contains default widget configuration
+// with any custom overrides defined in your admin settings.
+const config = OktaUtil.getSignInWidgetConfig();
```

Comment:
Check references to `config`. `OktaUtil` is only available in Okta-hosted scenario